project_generator/templates/makefile_gcc.tmpl

```diff
@@ -40,7 +40,7 @@ TARGET_EXT =
 TARGET_EXT = a
 {% endif %}
 
-LD_SCRIPT = {{linker_file}}
+LD_SCRIPT = {% for file in linker_file %} {{file}} {% endfor %}
```

Comment:
Can 2 linker command files be defined? It should trigger a warning if it's the case and pick up just the one